src/features/motivationList/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {
+  RefObject,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { FlatList, SafeAreaView, StatusBar, StyleSheet } from 'react-native';
+import { getUsers } from '../../utils/random-user';
+import { ListItem } from './ListItem';
+
+interface ListItemProps {
+  item: {
+    picture: {
+      large: string;
+      medium: string;
+      thumbnail: string;
+    };
+    name: {
+      title: string;
+      first: string;
+      last: string;
+    };
+  };
+}
+
+const _keyExtractor = (item: object, index: number) => index.toString();
+
+export const MotivationListScreen = () => {
+  const [users, setUsers] = useState([]);
+  const [page, setPage] = useState(1);
+  const [refreshing, setRefreshing] = useState(false);
+
+  const ref: RefObject<FlatList> = useRef(null);
+
+  useEffect(() => {
+    getUsers(20, 1, setUsers);
+  }, []);
+
+  const _onRefresh = async () => {
```

Comment:
No need to call the function with underscore. Usually, in JS it means pseudo-private fields/functions/variables, which is not the case for us.